src/Dfe.PlanTech.Domain/Questionnaire/Models/Section.cs

```diff
@@ -24,4 +24,14 @@ public class Section : ContentComponent, ISection
     public RecommendationPage GetRecommendationForMaturity(Maturity maturity)
      => Recommendations.FirstOrDefault(recommendation => recommendation.Maturity == maturity) ??
         throw new KeyNotFoundException($"Could not find recommendation with maturity {maturity}");
+
+    public RecommendationPage GetRecommendationForMaturity(string maturity)
+    {
+        if (maturity is null)
+            maturity = string.Empty;
+
+        var _ = Enum.TryParse(maturity, out Maturity maturityResponse);
```

Comment:
Would it be worth either removing the variable assignment (`_`), or checking if `Enum.TryParse` returns false and throwing an error?

Reply:
It's not even needed, it was just to remove that bloody code smell.  Even though it;s not picking up the checks upstream.